pkg/virt-controller/watch/drain/evacuation/evacuation_test.go

```diff
@@ -445,6 +446,89 @@ var _ = Describe("Evacuation", func() {
 
 			controller.Execute()
 		})
+
+		It("Should not evict new VMIs when the amount of concurrent running migrations reached its maximum capacity", func() {
+			var maxParallelMigrationsPerCluster uint32 = 5
```

Comment:
nit: how about turning this into a const?

Reply:
I will have to bump the pointer package first.